src/styles/blocks/page/_page.scss

```
@@ -0,0 +1,69 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 169%;
+  color: $pageColor;
+  scroll-behavior: smooth;
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    transform: translate(-100%, -100%);
+    transition: transform 0.3s;
```

If you have $duration variable, why don't you use it in every transition? Make sure to replace all the constant time definitions with this variable
```suggestion
    transition: transform $duration;
```